### PR TITLE
adding comments to publications.

### PR DESCRIPTION
**Pull request by cranmer**

### CODE REVIEW

File: _data/publications/1675782.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@ inspire-id: 1675782
 project: 
 - madminer
 - exploratory-ml
-link: https://doi.org/10.1073/pnas.1915980117
+link: "https://doi.org/10.1073/pnas.1915980117"
```
Comment (henryiii) [inline _data/publications/1675782.yml:5]: Quotes are optional. By design, URLs are always valid implicit strings in YAML. The only strings that must have quotes are ones with `: ` (colon followed by space), and other valid yaml identifiers (a single number, `true`, etc.)

Comment (cranmer) [reply _data/publications/1675782.yml:5]: thanks, I was a little confused when the : needed a "". I also have `DOI:10.10/...`. so put quotes on those as well. It's the space that I was missing.

Comment (cranmer) [reply _data/publications/1675782.yml:5]: I merged this myself b/c Peter says it's urgent

Comment (henryiii) [reply _data/publications/1675782.yml:5]: That's fine! This is a proceedings & slides day for me...

**Pull request merged**